SEOImages: User Story 1463880, Part 34 (#11426)
dotnet/docs: docs/framework/winforms/advanced/how-to-tile-a-shape-with-an-image.md

  
-   This example uses the following 75×75 image to tile a 200×200 rectangle.  
  

>  ![Screenshot that shows a tile image.](./media/how-to-tile-a-shape-with-an-image/rectangle-tile-200x200.gif)  

Reviewer: ```suggestion
 ![The tile image that shows a red house and a tree.](./media/how-to-tile-a-shape-with-an-image/rectangle-tile-200x200.gif)  
```

  
-   The following illustration shows how the rectangle is tiled with the image. Note that all tiles have the same orientation; there is no flipping.  
  

>  ![Screenshot of a rectangle tiled with an image.](./media/how-to-tile-a-shape-with-an-image/rectangle-tiled-image-no-flip.gif)  

Reviewer: ```suggestion
 ![A rectangle tiled with the image using the same orientation for all tiles.](./media/how-to-tile-a-shape-with-an-image/rectangle-tiled-image-no-flip.gif)  
```

  
-   This example uses the same 75×75 image to fill a 200×200 rectangle. The wrap mode is set to flip the image horizontally. The following illustration shows how the rectangle is tiled with the image. Note that as you move from one tile to the next in a given row, the image is flipped horizontally.  
  

>  ![Screenshot that shows a tile flipped horizontally.](./media/how-to-tile-a-shape-with-an-image/rectangle-tiled-image-horizontal-flip.gif)  

Reviewer: ```suggestion
 ![A rectangle tiled with the image flipped horizontally.](./media/how-to-tile-a-shape-with-an-image/rectangle-tiled-image-horizontal-flip.gif)  
```

  
-   This example uses the same 75×75 image to tile a 200×200 rectangle. The wrap mode is set to flip the image both horizontally and vertically. The following illustration shows how the rectangle is tiled by the image. Note that as you move from one tile to the next in a given row, the image is flipped horizontally, and as you move from one tile to the next in a given column, the image is flipped vertically.  
  

>  ![Screenshot that shows a tile flipped horizontally and vertically.](./media/how-to-tile-a-shape-with-an-image/rectangle-tiled-image-horizontal-vertical-flip.gif)  

Reviewer: ```suggestion
 ![A rectangle tiled with the image flipped horizontally and vertically.](./media/how-to-tile-a-shape-with-an-image/rectangle-tiled-image-horizontal-vertical-flip.gif)  
```
